webpack/scenes/ContentViews/Details/Promote/ContentViewVersionPromote.js

```diff
@@ -112,28 +122,34 @@ const ContentViewVersionPromote = ({
               onChange={value => setDescription(value)}
             />
           </FormGroup>
-        </Form>
-        <hr />
-        {forcePromote.length > 0 && (
-          <Alert variant="info" isInline title="Force Promotion">
-            <p>{__('Selected environments are out of the environment order. ' +
-              'The recommended practice is to promote to the next environment in the path.')}
-            </p>
-            <ComponentEnvironments environments={forcePromote} />
-          </Alert>)}
-        <EnvironmentPaths
-          userCheckedItems={userCheckedItems}
-          setUserCheckedItems={setUserCheckedItems}
-          promotedEnvironments={versionEnvironments}
-          publishing={false}
-        />
-        <Form>
-          <ActionGroup>
+          {!alertDismissed && forcePromote.length > 0 && (
+            <Alert
+              variant="info"
+              isInline
+              title={__('Force promotion')}
+              actionClose={<AlertActionCloseButton onClose={() => setAlertDismissed(true)} />}
+            >
+              <TextContent>
+                {forcePromote.length > 1 ? __('Selected environments ') : __('Selected environment ')}
+                <ComponentEnvironments environments={forcePromote} />
+                {forcePromote.length > 1 ?
+                  __(' are out of the environment path order. The recommended practice is to promote the next environment in the path.') :
```

Comment:
```suggestion
                  __(' are out of the environment path order. The recommended practice is to promote to the next environment in the path.') :
```

Reply:
Will add this to next ticket as there are a number of text changes there, and we can get this merged without waiting on CI.